Specs/DateTime/Dutch/DateTimeModel.json

```diff
@@ -19986,5 +19986,36 @@
         }
       }
     ]
+  },
+  {
+    "Input": "Ik ben afwezig november 19-20",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupportedByDesign": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "november 19-20",
+        "Start": 15,
+        "End": 28,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "(XXXX-11-19,XXXX-11-20,P1D)",
```

Comment:
Resolution is incorrect? Should be P2D start in 19 (inclusive) and ends in 21 (not inclusive).

Reply:
Is this case different from some other behaviour? We may have a breaking problem here.

Reply:
I believe it's another big topic - the date range resolutions are messed up, 
there is no clear rule when the last day/week/month is inclusive or not.

Reply:
This is how the parsing of this pattern is currently implemented. 
There are similar cases in the specs that follow the same logic, for example:
- "I'll be out from 4-23 in next month" -> (2016-12-04,2016-12-23,P19D)
- "I'll be out between 3 and 12 of Sept hahaha" -> (XXXX-09-03,XXXX-09-12,P9D)
- "APEC will happen in Korea on November 18-19" -> (XXXX-11-18,XXXX-11-19,P1D)
- "9 to 12 of June: another tapa festival" -> (XXXX-06-09,XXXX-06-12,P3D)

Reply:
I see. Some could be considered ambiguous, but some seem clearly incorrect. Let's not modify for now though, as this is somewhat of a breaking change.